python/singa/tensor(test).py

```diff
@@ -0,0 +1,1109 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
```

Comment:
did you add a new file? tensor(test).py
you can add a test_autograd.py under test/python